scripts/delta.sbatch.sh

```diff
@@ -33,11 +33,11 @@ srun_cmd="srun -N $nnodes -n $nproc"
 # See
 # https://mirgecom.readthedocs.io/en/latest/running.html#avoiding-overheads-due-to-caching-of-kernels
 # on why this is important
-export XDG_CACHE_HOME_ROOT="/tmp/$USER/xdg-scratch/rank"
+export XDG_CACHE_HOME_ROOT="$(pwd)/.mirge-cache/xdg-cache/rank"
 
 # Fixes https://github.com/illinois-ceesd/mirgecom/issues/292
 # (each rank needs its own POCL cache dir)
-export POCL_CACHE_DIR_ROOT="/tmp/$USER/pocl-cache/rank"
+export POCL_CACHE_DIR_ROOT="$(pwd)/.mirge-cache/pocl-cache/rank"
```

Comment:
Should this use `MIRGE_CACHE_ROOT`? (Same question for the other batch submitter scripts.)

Reply:
These scripts are supposed to be standalone, so I think not?

Reply:
Ah, gotcha.

Reply:
I think the batch script inherits the submission env, so could be a nice addtion here...